protocols/datagram.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+// TODO:
+// Come up with a better name for this.
+package beavertail;
+
+message DatagramPush {
+  string busID                    = 10;
+  uint64 passengerCount           = 20;
```

Comment:
Nitpick
```suggestion
  int32 passengerCount            = 20;
```

Reply:
I like the idea of shrinking the bit count down to 32 bits. However I don't think passengerCount should include negative nums, what do y'all think @kev231998 @LexiKat @kev231998 ?

Reply:
Yikes, I meant uint32